src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -390,6 +392,32 @@ export default {
         currentPage() {
             return Shopware.Store.get('cmsPage').currentPage;
         },
+
+        measurementDisplayUnitRepository() {
```

Comment:
displayUnitRepository

Reply:
I would terminate this since this https://github.com/shopware/shopware/pull/9559#discussion_r2095750799